esp-bootloader-esp-idf/Cargo.toml

```diff
@@ -37,3 +38,6 @@ log = ["dep:log"]
 
 ## Enable support for `defmt`
 defmt = ["dep:defmt"]
+
+## Enable support for OTA partition manipulation.
+ota = ["dep:crc"]
```

Comment:
Why is this optional?

Reply:
we can make it non-opt for sure ... it's one dependency less (but if it's not optional we can remove the metadata stuff)

Reply:
I think the only thing we need is the offset of the partition table. The firmware should be smart enough to then parse it, figure out if there are ota partitions, and work accordingly or issue an error. All of that is cut out if unused.

Reply:
actually I was more into the metadata stuff and have tools check the partition table - if we agree it's not worth it I'm happy to remove the feature